src/resolver.rs

```diff
@@ -207,12 +208,15 @@ fn python_files_in_path(
     }
 
     // Collect all Python files.
-    let files: Vec<Result<DirEntry, walkdir::Error>> = WalkDir::new(path)
+    let files: Vec<Result<DirEntry, ignore::Error>> = Walk::new(path)
         .into_iter()
-        .filter_entry(|entry| {
```

Comment:
I want to use `WalkBuilder`, which should be pretty much a drop-in replacement for `WalkDir` (and let's me use the same `filter_entry`), but it's requiring that the lifetime of any variable in the closure if `'static`? So I can't iteratively build up the shared `resolver` like I am now.

Reply:
The current code could be super slow if you had a very large directory that wasn't marked as `.gitignore` but was excluded by Ruff. E.g., if you had a monorepo with a bunch of directories that contained JS files, we'd spend time traversing all of those.